docs/index.md

```diff
@@ -0,0 +1,476 @@
+        (x) =>
+        {
+            this.ViewPoint = (Viewpoint)x;
+        }, true));
+    }
+}
+```
+
+# App architecture
+
+The app is built using the [MVVM Pattern](https://msdn.microsoft.com/en-us/library/hh848246.aspx). The UI is contained in the app's Views, while the business logic is stored in the ViewModels and the objects are stored in Models. The app makes use of properties, bindings and commands to pass information between the views and view models.
+
+The tablet-like UI and Navigation is not typical to a desktop WPF application but was created to support both desktops and tablet devices with touch screens. The app is structured as one container with multiple child views that get switched based on the user's interaction with the app.
+
+```xml
+<Window.Resources>
+	<DataTemplate DataType="{x:Type viewmodels:DownloadViewModel}">
+		<local:DownloadView />
+	</DataTemplate>
+	<DataTemplate DataType="{x:Type viewmodels:MainViewModel}">
+		<local:MainView />
+	</DataTemplate>
+	<DataTemplate DataType="{x:Type viewmodels:MapViewModel}">
+		<local:MapView />
+	</DataTemplate>
+</Window.Resources>
+
+<!-- View model for the active user control -->
+<ContentControl Content="{Binding DisplayViewModel}"/>
+```
+
+When the app first loads, a singleton instance of the `AppViewModel` is created and the `DownloadView` is set as the active view.
+
+```csharp
+// Test if AppViewModel singleton instance exists
+if (AppViewModel.Instance == null)
+{
+	// Set data context for the main screen and load main screen
+	AppViewModel.Instance = AppViewModel.Create();
+	this.DataContext = AppViewModel.Instance;
+}
+
+// Make instance of the DownloadViewModel and set it as datacontext.
+// This will set the active view as the DownloadView
+try
+{
+	var downloadViewModel = new DownloadViewModel();
+	AppViewModel.Instance.DisplayViewModel = downloadViewModel;
+	await downloadViewModel.ConnectToPortalAsync();
+}
+```
+
+When the Mobile Map Package has been downloaded, the app switches the active view `AppViewModel.Instance.DisplayViewModel = new MainViewModel();`
+
+# App customization
+
+Since all settings are stored in a config file, modifying the app to work with a different mobile map package only requires some XML changes. Open the `OfflineMapBook.exe.config` file located in the `bin\Release` folder and change some or all of the settings listed below to fit your needs. These changes will take effect immediately and will not require a recompile of the app.
+
+## Customizable
+
+### `MmpkFileName`
+
+The name of the mobile map package as it appears in ArcGIS Online. This will include the .mmpk extension.
+
+### `PortalUri`
+
+The URL of the sharing REST endpoint ArcGIS Online or Portal instance that stores the mobile map package. Example, if you access the mmpk in the browser by going to
+
+> http://runtime.maps.arcgis.com/home/item.html?id=219e6bc23dc64b3696ab6bc4f09deadb
```

Comment:
I'm not sure what mmpk this was ever referencing, but it no longer exists. It should be this one instead:
Verified here: https://github.com/Esri/mapbook-wpf/blob/2d5ad6358345c7168551f92a68ed2913b77b38c2/OfflineMapBook/OfflineMapBook/app.config#L28

```suggestion
> https://arcgisruntime.maps.arcgis.com/home/item.html?id=b5d32c90c9384187b4b8f041bd698c42
```